src/components/Alerts/SolanaAlert.tsx

```diff
@@ -6,19 +6,10 @@ import { useChainTokenSelectionStore } from 'src/stores';
 
 export const SolanaAlert = () => {
   const { t } = useTranslation();
-  const { sourceChainToken, destinationChainToken } =
-    useChainTokenSelectionStore();
-
-  const solanaSelected = useMemo(() => {
-    const isSelected =
-      sourceChainToken?.chainId === ChainId.SOL ||
-      destinationChainToken?.chainId === ChainId.SOL;
-    return isSelected;
-  }, [destinationChainToken, sourceChainToken]);
 
   return (
     <InfoAlert
-      active={solanaSelected}
+      active={true}
```

Comment:
I don´t get the whole picture, yet. But if we went on like that, you might as well remove the "active" prop at all. But let´s have a chat.